### PR TITLE
Demo User Buttons

### PR DESCRIPTION
**Pull request by jerixmx**
### What this PR does (required):
- Adds "Use Demo" Buttons to Login and Signup Page
- Defines demo user credentials
- Sets user details to the demo user credentials
- Routes user to dashboard

### Screenshots / Videos (required):
- https://user-images.githubusercontent.com/78225194/125495698-b9e31dc0-eda2-4571-b902-e381474660d5.mp4

### Any information needed to test this feature (required):
- Click `Use Demo` in the signup and login page
- Use demo credentials (in `/client/src/helpers/demovalues.ts`) in the login page

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

File: client/src/helpers/demovalues.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const demoValues = {
```
Comment (rajivtitus) [inline client/src/helpers/demovalues.ts:1]: 👍🏽

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 2]
```diff
@@ -31,6 +34,16 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  let demo = false;
+  const demoLogin = () => {
+    login(demoValues.email, demoValues.password).then((data) => {
+      if (data.success) {
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx]: If there is an error maybe we can update the snack bar (useSnackbarContext) to display the error message ? :)

**Review by rajivtitus — APPROVED** [review]
Good work! :) 

Just one small comment, please have a look.

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx (hunk 2 above)
Comment (jerixmx) [reply client/src/pages/SignUp/SignUpForm/SignUpForm.tsx]: Will add, thanks!

**Pull request merged**